wiki/content/graphql/queries/persistent-queries.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "Persistent Queries"
+weight = 3
+[menu.main]
+    parent = "graphql-queries"
++++
+
+Dgraph supports Persistent Queries. When persistent queries are enabled, the client only sends the hash of a query to the server. The server has a list of known hashes and uses the related query accordingly. 
```

Comment:
There is no work required to enable persistent queries for dgraph per-se. If the input request from clients like Apollo or HTTP request with appropriate extenstions mention `perstistentQuery` then the query will be persisted. There is no `enabling` required from servers running dgraph.

Reply:
Fixed